homeassistant/components/media_player/__init__.py

```diff
@@ -133,6 +145,27 @@
     vol.Optional(ATTR_MEDIA_ENQUEUE): cv.boolean,
 })
 
+MEDIA_PLAYER_PLAY_SONG_SCHEMA = MEDIA_PLAYER_SCHEMA.extend({
+    vol.Required(ATTR_MEDIA_SONG_NAME): cv.string,
+    vol.Optional(ATTR_MEDIA_ARTIST_NAME): cv.string,
+})
+
+MEDIA_PLAYER_ADD_SONG_SCHEMA = MEDIA_PLAYER_SCHEMA.extend({
+    vol.Optional(ATTR_MEDIA_SONG_ID): cv.string,
+    vol.Optional(ATTR_MEDIA_SONG_NAME): cv.string,
+    vol.Optional(ATTR_MEDIA_ARTIST_NAME): cv.string,
```

Comment:
No this should work also without anything, even though the output would be simply the first song in the library.